exercises/allergies/test/Tests.hs

```diff
@@ -77,7 +77,7 @@ specs = do
 
             it "rejects multiple mismatching allergens" $
               forAllShrink complementsWithScore shrinkComplementsWithScore $
-                \(allergens, score) -> all (not . (`isAllergicTo` score)) allergens
+                \(allergens, score) -> not (any (`isAllergicTo` score) allergens)
```

Comment:
I'm remarking that this change was made necessary by the lts-14.7 -> lts-15.5 update because a newer version of HLint comes with more linting rules.